egs2/marathi_lrec2020/asr1/README_multitask.md

```diff
@@ -0,0 +1,205 @@
+# Automatic Speech Recognition (Multi-tasking)
```

Comment:
I do not see the necessity of this readme. If there is no additional reason, I would recommend we safely remove this readme.